Dockerfile.windows

```diff
@@ -0,0 +1,13 @@
+# escape=`
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG PIXI_VERSION
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
is changing the shell necessary? I would prefer to keep the images as close to stock as possible

Reply:
agreed